package.json

```diff
@@ -7,8 +7,10 @@
     "dompurify": "^1.0.11",
     "lodash": "^4.17.15",
     "moment": "^2.24.0",
+    "moment-timezone": "^0.5.26",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
+    "react-moment": "^0.9.2",
```

Comment:
@VivianDoreen There was no need to add these extra dependencies since we already have moment installed.